main.py

```diff
@@ -49,6 +51,8 @@ def getNeighbors():
 FLOAT_LAT = 0
 FLOAT_LONG = 0
 
+mymap = pygmaps.pygmaps(38.8855018,-77.1796793,15.08)
```

Comment:
Could you get these co-ordinates from the origin instead of hard-coding your location?